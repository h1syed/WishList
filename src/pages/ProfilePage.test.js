import React from "react";
import { shallow } from "enzyme";
import ProfilePage from "./ProfilePage";

describe("ProfilePage", ()=>{
    const profilepage = shallow(<ProfilePage />);

    it("renders properly", () => {
        expect(profilepage).toMatchSnapshot();
    });

   

    
});