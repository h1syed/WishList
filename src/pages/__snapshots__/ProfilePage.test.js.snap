// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProfilePage renders properly 1`] = `ShallowWrapper {}`;
